spring-data-vmware-gemfire/build.gradle

```diff
@@ -165,4 +175,16 @@ String getGemFireBaseVersion() {
   return "${split[0]}.${split[1]}"
 }
 
+tasks.register('copyJavadocsToBucket') {
+  dependsOn javadocJar
+  doLast {
+    Storage storage = StorageOptions.newBuilder().setProjectId(project.findProperty('docsGCSProject')).build().getService()
+    BlobId blobId = BlobId.of(project.findProperty("docsGCSBucket"), "${project.findProperty('pomProjectArtifactName')}/${project.version}/${javadocJar.outputs.files.singleFile.name}")
+    BlobInfo blobInfo = BlobInfo.newBuilder(blobId).build()
+    storage.createFrom(blobInfo, javadocJar.outputs.files.singleFile.toPath())
+  }
+}
 
+publish {
+  finalizedBy copyJavadocsToBucket
+}
```

Comment:
Is this going to run EVERY build we do?

Reply:
If we run publish, yes–but it shouldn't run just from running `gradle build`

Reply:
Is that not the behavior we want? I figured we'd want it to automatically do any newly released artifacts.

Reply:
would this run on even normal builds or is smart enough to know when we are releasing?

Reply:
Do we run `publish` other than when we release? I ran a normal build locally and it didn't push anything, even with the parameters set. It only triggers when the `publish` command is run.

Reply:
Hmm, actually, you might be right. We do run publish in all pipeline builds, we just publish to our artifact repo, not JFrpg. I'll change this logic.